src/os_auth/local-server.c

```diff
@@ -342,6 +342,7 @@ char* local_dispatch(const char *input) {
     output = cJSON_PrintUnformatted(response);
     cJSON_Delete(response);
     cJSON_Delete(request);
+    os_free(groups);
```

Comment:
We can avoid changing the scope of groups if we change the parsing order of the force block. Leaving groups as the last parsed element, let us keep one unique free and keep groups in the original scope.